styleguide/_themes/global/scss/fonts.scss

```diff
@@ -2,11 +2,13 @@ $heading: 'Khand', Helvetica, Arial, sans-serif;
 $subhead: 'Khand', Helvetica, Arial, sans-serif;
 $copy: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
 $code: Tahoma, Arial, Geneva, Helvetica;
+$font-rsweb: 'rsweb';
```

Comment:
@writes 
Why add in the variable here? It looks like you are declaring it here but not using it.